### ISSUE TITLE
Make action list UI resizeable by user

### ISSUE THREAD

**Issue by MartinHaeusler** [gaearon/redux-devtools#373]
I really love the power and simplicity of the redux dev tools. There's just one little thing that bugs me about it. I'm the kind of guy who has very looooong names for redux actions:

![image](https://user-images.githubusercontent.com/16356998/27804685-91a3ebd8-6030-11e7-845d-6b9ae917cc17.png)

The problem is that the action names are cut off, and the list view that displays them cannot be resized horizontally by the user. The only workaround I am aware of is to make the entire window full-size, such that the proportion of space assigned to the list becomes big enough to hold the contents. 

Can you do anything about this? If the vertical separator between the list view and the state/action view would be draggable (like a split pane), I'd be very happy. I'm using the redux dev tools on windows 10 and chrome.

**Comment by codercodingthecode** [comment]
This is my own custom build. 
I was tired of not being able to see the full picture of the data, I care more about the data than the updates on the left most of the time.
From what I read, they really don't want to do it, otherwise, I will be happy to open a pr, 
If anyone else wants it, it will be on my GH.
I will be adding some extra features over the course of the upcoming days.
Like a snap-in button for instance.

![Kapture 2022-04-15 at 22 46 52](https://user-images.githubusercontent.com/20182536/163660333-7e1547bd-2d32-460a-97f7-af94a16e9003.gif)

I have requested this feature over react devtools https://github.com/facebook/react/issues/17389
and it was done. I don't know why it's not in here yet though.
Seems like a pretty handy feature to have.

**Comment by VeikkoLehmuskorpi** [comment]
> From what I read, they really don't want to do it, otherwise, I will be happy to open a pr,
If anyone else wants it, it will be on my GH.

Any links to those discussions, perhaps in other issues?

Did you ever fork this? I couldn't find the repo from your profile.

**Comment by codercodingthecode** [comment]
> 

Yes, it is in my GitHub. I will share this work repo this afternoon after work.

**Comment by muescha** [comment]
i also missed this feature to resize the devpanel overlay because the values are not visible. the panel is only a 1/4 of my screen width.
